zombienet/tests/0001-asset-transfer-works-westend-to-rococo.zndsl

```diff
@@ -22,5 +22,12 @@ asset-hub-rococo-collator1: js-script ../helpers/wrapped-assets-balance.js with
 bridge-hub-rococo-collator1: js-script ../helpers/relayer-rewards.js with "5FLSigC9HGRKVhB9FiEo4Y3koPsNmBmLJbpXg2mp1hXcS59Y,0x00000002,0x62687764,BridgedChain,0" within 300 seconds
 bridge-hub-rococo-collator1: js-script ../helpers/relayer-rewards.js with "5FLSigC9HGRKVhB9FiEo4Y3koPsNmBmLJbpXg2mp1hXcS59Y,0x00000002,0x62687764,ThisChain,0" within 300 seconds
 
+# step 7: send wWND back to Alice at Westend AH
```

Comment:
@svyatonik what about adding here some check that user already has som wWND?

this file is 0001-asset-transfer-works-**westend-to-rococo**.zndsl but we start on AssetHubRococo :)

Reply:
> @svyatonik what about adding here some check that user already has som wWND?

It is tested above:
```
# step 5: elsewhere Westend has sent WOC to //Alice - let's wait for it
asset-hub-rococo-collator1: js-script ../helpers/wrapped-assets-balance.js with "5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY,0,Westend" within 600 seconds
```

> this file is 0001-asset-transfer-works-westend-to-rococo.zndsl but we start on AssetHubRococo :)

Yeah, I know - it causes confusion :) I could move to sibling file, but since transaction is made at Rococo AH, I thought it is better to keep it here. So all scripts/tests that are checking Rococo parachains are happening here and Westend are happening in sibling file. Open to suggestions though

Reply:
I've updated comments here: https://github.com/paritytech/parity-bridges-common/pull/2711/commits/e31ae6b7e6f54f331f22625bc4eb491212823c0c

Yes, technically "step 4" belongs to other tests file, but let's keep all Rococo transaction submitter script calls in 0001-asset-transfer-works-westend-to-rococo.zndsl and Westend in 0001-asset-transfer-works-rococo-to-westend.zndsl. When Zombienet will support multiple relay chains in single network/test file, it should be clearer